### ISSUE TITLE
Update references to rtyler/jenkins in README

### ISSUE THREAD

**Issue by gguillotte** [voxpupuli/puppet-jenkins#934]
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: N/A
- Ruby: N/A
- Distribution: N/A
- Module version: master branch

## How to reproduce (e.g Puppet code you use)

N/A

## What are you seeing

    # Using puppet-jenkins

    ## Getting Started
    ```bash
    puppet module install rtyler/jenkins
    ```

etc.

## What behaviour did you expect instead

    # Using puppet-jenkins

    ## Getting Started
    ```bash
    puppet module install puppet/jenkins
    ```

## Output log

N/A

## Any additional information you'd like to impart

If this is just a search/replace operation in the README, I can file a PR. If there are other code or example changes necessary in the README, some testing would be required.

**Comment by ghoneycutt** [comment]
Thanks for reporting this. Could you send us a PR with your changes and add a comment with `Fixes #934`

**Comment by gguillotte** [comment]
@ghoneycutt done, thanks!
